### ISSUE TITLE
Training output perform much worse than pre-train segnet_basic_camvid model  

### ISSUE THREAD

**Issue by miketry0313** [alexgkendall/SegNet-Tutorial#83]
Hi All,

I am fine tuning `segnet_basic_camvid.caffemodel` on Cityscape data by using `segnet_basic_train.prototxt` and `segnet_basic_solver.prototxt`. The output of loss and avg accuracy looks good (loss: 0.76, avg accuracy: 0.6). But when I using `test_segmentation_camvid.py` to display inference model result, it looks much worse than pre-train model. Can anyone give me some help in advance? 

![ground_truth](https://cloud.githubusercontent.com/assets/8083326/23476489/c056faba-fe6f-11e6-95a1-fb0c3dc689b1.png)
                                         (Ground Truth)

![infer_result_pretrain_model](https://cloud.githubusercontent.com/assets/8083326/23476497/c7ed405e-fe6f-11e6-8af5-a9bda0dde510.png)
                                        ( Pretrain Model Inference Result)

Below are the result of inference model before calculating bn statistics and after bn statistics.
![infer_result_before_compute_stat](https://cloud.githubusercontent.com/assets/8083326/23476875/fcec7878-fe70-11e6-99fc-e569c8385ab0.png)

![infer_result_after_compute_stat](https://cloud.githubusercontent.com/assets/8083326/23476876/ffdf2422-fe70-11e6-9151-f0fad6c70b7f.png)
